### ISSUE TITLE
Story: Match Making: Peer-to-peer

### ISSUE THREAD

**Issue by z-schwartz** [IITIDIDX597/F2021_Team2#20]
**PROJECT GOAL**
Community Catalyst connects a resident in the community with partners that help transform the collective vision for the neighborhood into action ensuring a more equitable and sustained community recovery.

**THE REQUISITE SUB-HILL STATEMENT**
Grouping: Match Making
1. Volunteer can access an online directory of alumni so that they can interact and receive leadership development advice. 
2. The volunteer can see a "good chemistry" feature that helps them match with fellow volunteers
3. The volunteer can find a mentor matched to their specific user profile and dashboard data 
4. From their dashboard, volunteers can see which upskill power-ups and incentives are associated to current volunteering opportunities.

**THE STORY FOR REVISION**
**As a:** volunteer

**I want:** my profile to produce a matching matrix visible to other volunteers (on my permission)

**So that:** other volunteers with similar interest can find me and reach out to me as well

**Comment by z-schwartz** [comment]
- [x] @gauribhatt - as discussed, please refine 2 more times before our 5 o'clock meeting.

**Comment by gauribhatt** [comment]
Iteration  1 -

As a: volunteer

I want: the platform to pair me volunteers based on our common interests

So that: we are compatible while working together

**Comment by gauribhatt** [comment]
**Iteration 2 -** 

**As a:** volunteer

**I want:** to be able to view a volunteer compatibility matrix on my profile

**So that:** I can be matched with like-minded volunteers

**Comment by z-schwartz** [comment]
**Iteration 3** 

As a: volunteer

I want: to be able to view my compatibility with other volunteers

So that: I can be matched with like-minded volunteers

**Comment by akallish** [comment]
I like the second iteration because the "I want" statement feels more powerful to see compatible profiles when viewing your own.

**Comment by z-schwartz** [comment]
> **Iteration 2 -**
> 
> **As a:** volunteer
> 
> **I want:** to be able to view a volunteer compatibility matrix on my profile
> 
> **So that:** I can be matched with like-minded volunteers

Agreed @akallish. This one is best as it's more specific to the "what" and hints at how it might look.

**Comment by akallish** [comment]
> Agreed @akallish. This one is best as it's more specific to the "what" and hints at how it might look.

@z-schwartz Good

**Comment by Tosgirl** [comment]
@gauribhatt iteration 1, @z-schwartz iteration 2

**Comment by z-schwartz** [comment]
@gauribhatt @Tosgirl 

Can we switch this so that I am iteration 1? I don't have an iteration one otherwise.

**Comment by z-schwartz** [comment]
### STORY
> **As a:** volunteer
> 
> **I want:** to be able to view a volunteer compatibility matrix on my profile
> 
> **So that:** I can be matched with like-minded volunteers
> 

### ACCEPTANCE CRITERIA
**Given:** I'm signed in on the platform and am seeing my profile dashboard

**When:** I'm able to see how other residents in my community and how compatible I am with them based on past experiences, current skills, and preferences

**Then:** I or they can reach out to connect so we can begin projects together

**Comment by z-schwartz** [comment]
@gauribhatt this is ready for you to iterate on.

**Comment by gauribhatt** [comment]
ACCEPTANCE CRITERIA Iteration 2

Given: I'm a volunteer on the CC platform viewing my profile on the dashboard
When: I'm able to see other volunteers in my community and my compatibility with them with the help of an algorithm that is based on our profiles
Then: we can reach out to each other to connect and work together

**Comment by gauribhatt** [comment]
ACCEPTANCE CRITERIA Iteration 3

Given: I'm a volunteer on the CC platform viewing my profile on the dashboard
When: I'm able to see other volunteers in my community and my compatibility with them with the help of an algorithm that matches us based on our profile characteristics such as:
- past experiences
- current skills
- preferences
Then: we can start a conversation with each other on the platform after we have mutually accepted a match, to connect and work together

**Comment by akallish** [comment]
@gauribhatt good iteration. Cc shoul provide % matches based on criteria

**Comment by gauribhatt** [comment]
Oh I like the idea about % matches!

**Comment by gauribhatt** [comment]
ACCEPTANCE CRITERIA Iteration 4

Given: I'm a volunteer on the CC platform viewing my profile on the dashboard
When: I'm able to see other volunteers in my community and my % compatibility with them on different parameters with the help of an algorithm that matches us based on our profile characteristics such as:
- past experiences
- current skills
- preferences
Then: we can start a conversation with each other on the platform after we have mutually accepted a match, to connect and work together

**Comment by akallish** [comment]
> Then: we can start a conversation with each other on the platform after we have mutually accepted a match, to connect and work together

Then: I choose the people matches that are most meaningful to me to connect for possible collaboration.

@gauribhatt

**Comment by gauribhatt** [comment]
Perfect! Thanks @akallish 

ACCEPTANCE CRITERIA Iteration 5

Given: I'm a volunteer on the CC platform viewing my profile on the dashboard
When: I'm able to see other volunteers in my community and my % compatibility with them on different parameters with the help of an algorithm that matches us based on our profile characteristics such as:
-past experiences
-current skills
-preferences
Then: I choose the people matches that are most meaningful to me to connect for possible collaboration.

**Comment by z-schwartz** [comment]
@gauribhatt - looks good!
